cluster-up/cluster/kind/common.sh

```diff
@@ -153,9 +182,11 @@ function setup_kind() {
     done
 
     echo "ipv6 cni: $IPV6_CNI"
-    if [ -z ${IPV6_CNI+x} ]; then
+     if [ -z ${IPV6_CNI+x} ]; then
```

Comment:
Why this changed?

Reply:
My bad, I removed this check on the next commit anyway